ContiguousArray.java

```diff
@@ -0,0 +1,31 @@
+class Solution {
+    public int findMaxLength(int[] nums) {
+        
```

Comment:
Good Job! Please add time and space complexities and comments explaining your approach.